### ISSUE TITLE
Array CollectionForm

### ISSUE THREAD

**Issue by CarlosUvaSilva** [rswag/rswag#278]
```
parameter name: :with, in: :query, schema: { type: :array, collectionFormat: 'multi',
        items: { type: 'string', enum: ['recipes', 'ingredients'] }}, allowReserved: true
```

I'm trying to get a URL like `?with[]=recipes&with[]=ingredients` but I always get `?with=recipes&with=ingredients`. This obviously doesn't work with Rails because I if do `params[:with]` I only get the last one (ingredients). I've tried to change collectionFormat to "pipes" just to test but I'm never able to change the format of the URL

**Comment by BookOfGreg** [comment]
Hi @CarlosUvaSilva ,
Looks like you have the same issue as #256
If it gets answered, it will be answered there first.

**Comment by Kutomore** [comment]
Hi @CarlosUvaSilva the issue was solved on #256 if it's still relevant you might want to check it there.

**Comment by jtannas** [comment]
Heyo - new volunteer here sorting through the issue backlog and came across this.
This looks resolved by the linked issue, but if you're running into problems feel free to open a new issue and I'll be happy to help :slightly_smiling_face: 
Cheers!
